code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -243,6 +243,8 @@
 	destination.underwear_color = underwear_color
 	destination.undershirt = undershirt
 	destination.socks = socks
+	destination.bra = bra //SKYRATS EDIT ADDITION
```

Comment:
You can override this modularly to add your thiings.

Also it's `SKYRAT` not `SKYRATS`, figured I'd say it now.